app/homepage.py

```diff
@@ -34,51 +34,49 @@
 nav = Navbar()
 
 body = dbc.Container(
-    [
-    dbc.Row(
+    [dbc.Row(
         [dbc.Col(html.H1('Bienvenue sur Pyronear')),
-        dbc.Col(dcc.Dropdown(id='user_department_input',
-                            options=[
-                            {'label': 'Ardèche', 'value': 'Ardèche'},
-                            {'label': 'Gard', 'value': 'Gard'},
-                            {'label': 'Landes', 'value': 'Landes'}
-                            ]))
-        ]),
-    dbc.Row(
+         dbc.Col(dcc.Dropdown(id='user_department_input',
+                              options=[{'label': 'Ardèche', 'value': 'Ardèche'},
```

Comment:
@nrslt  You will include later the logic that zoom on the department selected by the user?

Reply:
@VincGargasson definitely! As I explained yesterday, the container/row/column logic that is inherent of [dash bootstrap components](https://dash-bootstrap-components.opensource.faculty.ai/) is really practical and easy to remodel.
Here is what it looks like at the moment:
![Capture d’écran, le 2020-10-15 à 08 30 45](https://user-images.githubusercontent.com/64283717/96085537-40964900-0ec1-11eb-8716-1c1027548c08.png)
![Capture d’écran, le 2020-10-15 à 08 31 13](https://user-images.githubusercontent.com/64283717/96085563-45f39380-0ec1-11eb-8bdb-4b01decad4e5.png)

Reply:
@VincGargasson As for the department zoom, for now I just added the same map as alerts.py. We will need to add a callback element that links the user choice of department with a new map, centered on that department.

Again, the idea here is more about quickly setting a template for us to keep on working on the homepage (for @pechouc to work on the slider, which has its own available slot, and for you to work on the map)
Let me know if this logic makes sense and if you'd like any modification before approving this PR!

![Capture d’écran, le 2020-10-15 à 08 38 46](https://user-images.githubusercontent.com/64283717/96086214-417baa80-0ec2-11eb-8ae5-c5838e0e65bd.png)


cheers